### DISCUSSION TITLE
8bitdo N64 kit Does is not mapped correctly with Blueretro HW1

### DISCUSSION THREAD

**Discussion by Aleksi87** [darthcloud/BlueRetro#1097] [General]
### BlueRetro firmware version

24.10

### BlueRetro firmware specification

HW1

### BlueRetro firmware variant

Universal

### BlueRetro hardware type

Internal install DIY

### Manufacturer

Myself

### System used

Nintendo 64

### Bluetooth controller brand & name

8bitdo N64 kit

### What is problem? (only list ONE problem per report)

I have installed Blueretro HW1 inside of my Nintendo 64 using ESP32
I manage to SYNC my 8bitdo N64 Controller with the ESP32 but the only option as a SRC Controller is NSO N64 which I select.
After I do that I notice that the A and B buttons are not mapped at all or mapped on the opposite way.
When I SYNC my NSO N64 controller using the same configuration the buttons are mapped correctly without even setting the blueretro.io settings

### What did you expect to happen?

8bitdo N64 kit should work the same way as NSO N64 or create a new 8bitdo N64  mapper in a future FW update

**Comment by darthcloud** [comment]
Use N64 label (NOT NSO) as src and dst.

You need to connect in D mode for mapping to be correct by default.

https://github.com/darthcloud/BlueRetro/wiki/Controller-pairing-guide#15---8bitdo-n64-modkit-paring-guide
